src/services/shopping-cart.service.ts

```diff
@@ -1,21 +1,22 @@
 import { Injectable } from '@angular/core';
-import { of } from 'rxjs';
+import { Observable, of } from 'rxjs';
 import { Product } from 'src/interfaces/productInterface';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root',
 })
 export class ShoppingCartService {
+    products: Product[] = [];
 
-  products: Product[] = [];
-
-   addProductToCart(product: Product){
+    addProductToCart(product: Product): void {
         alert('You have added ' + product.productName + ' To your cart!');
```

Comment:
I think we can get rid of these alerts now, the cart number incrementing is sufficient for showing this is working

Reply:
I'm gonna try adding a toast